tests/test_serialize.cc

```diff
@@ -266,7 +288,7 @@ typedef ::testing::Types<LDLT, ExplainedCovarianceRepresentation, EigenMatrix3d,
                          EigenVectorXd, EmptyEigenMatrixXd, EigenMatrixXd,
                          FullJointDistribution, MeanOnlyJointDistribution,
                          FullMarginalDistribution, MeanOnlyMarginalDistribution,
-                         ParameterStoreType, Dataset, DatasetWithMetadata,
+                         Dataset, DatasetWithMetadata, ParameterStoreType,
```

Comment:
Ah, that was just left over from when I was commenting out and rearranging the order to debug some compile errors.  I'll revert it to avoid confusion.